lookout/style/format/benchmarks/top_repos_quality.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, quality: Optional[str] = None, model: Optional[str] = None) -
 
 
 def measure_quality(repository: str, from_commit: str, to_commit: str, port: int,
-                    review_config: dict, train_config: dict) -> QualityReport:
+                    review_config: dict, train_config: dict, bblfsh: str) -> QualityReport:
```

Comment:
Type annotation should still be Optional[str]

Reply:
oops, redundant space...